README.md

```diff
@@ -13,8 +13,8 @@ Add flutter_map to your pubspec:
 dependencies:
   flutter_map: any # or the latest version on Pub
 ```
-
-Configure the map using `MapOptions` and layer options:
+### Mapbox provider
+Configure the map to use [Mapbox](https://mapbox.com) by using the following `MapOptions` and layer options:
```

Comment:
We can't mention MapBox. See [this issue](https://github.com/johnpryan/flutter_map/issues/404) for details

Reply:
OK, i will revert this mention later today